### ISSUE TITLE
Can edit a Draft Election in a way that Election Settings does not appear.

### ISSUE THREAD

**Issue by awoflaherty** [TrustTheVote-Project/ABC-backend#127]
To reproduce:
1. Create a new test and put in test mode.
2. End Test Mode
3. From Dashboard, Election appears under Draft and then click on Continue Editing.  The Update Election page appears correctly.
4. **Rather than touch Next, use the Wizard feature beneath the Next button to select Election Settings.**
5. Election Settings appears with no values.
6. Realized you have "f*ed up", so click away and try again using the Next button. Values are still missing in Election Settings screen.
7. Enter new values in Election Settings screen, then click Next and all the data that was uploaded in Step 1 is gone.
8. Tiptoe out of this draft back to dashboard.  Appear from dashboard that EDF, ballots, etc is still intact, so put in test mode and use app to look at information.
9.  Realize that the changes you made in Election Settings on step 9 did not save. 
10. End Test Mode. Repeat steps 3 and 4.  Instead of step 5, use the Next button NOT the feature below the Next button.
11. Values from first test are there, so carefully edit, save and put into new Test mode.
12. Test mode works as expected.
In conclusion, I don't think the feature at the bottom of the page works consistently.

**Comment by trustthevote** [comment]
I agree that there not a good situation in the back-and-forth between (a) the sequence of next and back (b) random access via the numbered step sequence.

However, it is not a blocker for acceptance testing.

I would, however, like this issue to get a good look-at by Raynu, and determine the best way to streamline it.

**Comment by amekelburg** [comment]
Relates to https://github.com/TrustTheVote-Project/ABC-backend/issues/89
